tests/src/test/scala/whisk/core/limits/ActionLimitsTests.scala

```diff
@@ -248,7 +248,9 @@ class ActionLimitsTests extends TestHelpers with WskTestHelpers with WskActorSys
 
     // Needs some bytes grace since activation message is not only the payload.
     val args = Map("p" -> ("a" * (allowedSize - 750).toInt).toJson)
+    val start = Instant.now
```

Comment:
this change here looks unrelated to the rest of the PR.

Reply:
This test currently takes 15 seconds to invoke an action as serialization size is not account for even when the invoker truncates the action response.

```
whisk.core.limits.ActionLimitsTests > Action limits should successfully invoke an action with a payload close to the limit (5 MB) STANDARD_ERROR
    org.scalatest.exceptions.TestFailedException: 19204 was not less than 15000
```
```
whisk.core.limits.ActionLimitsTests > Action limits should succeed but truncate result, if result exceeds its limit (blocking: true) STANDARD_ERROR
    org.scalatest.exceptions.TestFailedException: 15754 was not less than 15000
```

Reply:
@rabbah, see above.